docs/developer/checkout/overview.mdx

```diff
@@ -249,8 +249,66 @@ As a result, we get an updated checkout object with the new checkout email set:
 
 ## Removing old checkouts
 
-To avoid overloading the database, unfinished checkouts are automatically deleted after a specified period from its last modification:
+To avoid overloading the database, unfinished & unpaid checkouts are automatically deleted after a specified period from its last modification:
```

Comment:
```suggestion
To avoid overloading the database, unfinished and unpaid checkouts are automatically deleted after a specified period from their last modification:
```